src/migration.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access */
+
+import { AccountID } from 'caip'
+import { fetchJson } from './utils'
+import type { BasicProfile } from '@ceramicstudio/idx-constants'
+import type { LinkProof } from '@ceramicnetwork/blockchain-utils-linking'
+
+const LEGACY_ADDRESS_SERVER = 'https://beta.3box.io/address-server'
+const THREEBOX_PROFILE_API = 'https://ipfs.3box.io'
```

Comment:
If we are on clay testnet, shouldn't we pull from the dev env of 3box?

Reply:
could, migration is flagged right now, so actually wont be deployed 

but even if deployed likely best to just use prod env, as people will actually have profiles to migrate, and it will get some testing before mainnet. It doesnt change any state in 3box or elsewhere of course, just in ceramic network, and once we switch to mainnet, nothing from clay should effect that. Also doesnt really add any load to 3box infra.

Reply:
I'm just afraid that it will confuse people if we enable it to think that they can start using the clay testnet for IDX. I'd rather have the separation be more clean here.
Actually, does develop.3box.io/ use the dev environment? If not I guess it will be hard to test.

Reply:
yeah develop.3box.io does not use dev env

another path, is deploy prod 3id-connect with no migration, and deploy dev 3id-connect with current configuration here (then we can all test internally easily at least)